programs/solvent-protocol/src/instructions/redeem_nft.rs

```diff
@@ -48,15 +48,57 @@ pub fn redeem_nft(ctx: Context<RedeemNft>, swap: bool) -> Result<()> {
         )?;
     }
 
-    // Send redeem// swap fee to Solvent treasury
-    let fee_amount = (DROPLETS_PER_NFT as u64)
+    // Calculate redeem/swap fee
+    let total_fee_amount = (DROPLETS_PER_NFT as u64)
         .checked_mul(LAMPORTS_PER_DROPLET as u64)
         .unwrap()
         .checked_mul(fee_basis_points as u64)
         .unwrap()
         .checked_div(10000)
         .unwrap();
-    let transfer_fee_ctx = CpiContext::new(
+
+    let distributor_fee_amount = (total_fee_amount as u64)
```

Comment:
A nitpick, but I think `total_fee_amount` is already `u64`, the cast is unneeded.